scripts/screeps_tencent_batch_rl_runner.py

```
@@ -0,0 +1,838 @@
+    parser.add_argument("--transfer-timeout-seconds", type=int, default=1200)
+    return parser
+
+
+def main(argv: list[str] | None = None) -> int:
+    args = build_parser().parse_args(argv)
+    args.preflight_only = args.command == "preflight"
+    run_id = args.run_id
+    artifact_dir = (REPO_ROOT / args.artifact_root / run_id).resolve()
+    controller = Controller(args=args, run_id=run_id, artifact_dir=artifact_dir)
+
+    def handle_signal(signum: int, _frame: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_frame`

(ANN401)